infrastructure/api_mtls.tf

```diff
@@ -0,0 +1,130 @@
+# New API Gateway for mTLS
+resource "aws_api_gateway_rest_api" "ndr_doc_store_api_mtls" {
+  name        = "${terraform.workspace}-DocStoreAPI-mTLS"
+  description = "Docuemnt store API with mTLS enabled"
+
+  tags = {
+    Name        = "${terraform.workspace}-docstore-api-mtls"
```

Comment:
Probably the same with the Name tag as with the resource name to keep it consistent. e.g. ndra_